cypress/e2e/gui/testsGui.cy.js

```diff
@@ -0,0 +1,358 @@
+describe('Validate the GUI', () => {
+  context('Validate the no customers scenarios', () => {
+    beforeEach(() => {
+      const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+      cy.intercept(
+        'GET',
+        `${CUSTOMERS_API_URL}?**`,
+        {
+          'customers': [],
+          'pageInfo': {
+            'currentPage': 1,
+            'totalPages': 1,
+            'totalCustomers': 50
+          }
+        }
+      ).as('getNoCustomers');
+      cy.setCookie('cookieConsent', 'accepted');
+      cy.visit('/');
+    });
```

Comment:
```suggestion
    });

```

Para melhorar a legibilidade do código, recomenda-se deixar uma linha em branco entre cada caso de teste (cada bloco `it`).

Reply:
O mesmo vale para outros testes.